packages/@okta/vuepress-site/docs/reference/api/factors/index.md

```diff
@@ -449,6 +449,163 @@ curl -v -X GET \
 ]
 ```
 
+### List YubiKey OTP Tokens
+
+<ApiOperation method="get" url="/api/v1/org/factors/yubikey_token/tokens" />
+
+Enumerates all YubiKey OTP Tokens
+
+##### Request parameters
+
+| Parameter | Description                                                                                                      | Param Type | DataType | Required | Default |
+| --------- | ---------------------------------------------------------------------------------------------------------------- | ---------- | -------- | -------- | ------- |
+| after     | Specifies the pagination cursor for the next page of tokens                                                      | Query      | String   | FALSE    |         |
+| expand    | If YubiKey Token is assigned to a user and expand is set to `user`, embeds the [user](/docs/reference/api/users/#user-object) resource   | Query      | String   | FALSE    |         |
```

Comment:
Switch the order of this to:

Embeds the [User](/docs/reference/api/users/#user-object) resource if the YubiKey Token is assigned to a user and expand is set to `user`